src/main/scala/tech/cryptonomic/conseil/tezos/TezosTypes.scala

```diff
@@ -233,7 +233,12 @@ object TezosTypes {
   final case object DoubleEndorsementEvidence extends Operation
   final case object DoubleBakingEvidence extends Operation
   final case object Proposals extends Operation
-  final case object Ballot extends Operation
+  final case class Ballot(ballot: Vote) extends Operation
+
+  sealed trait Vote extends Product with Serializable
```

Comment:
I might try to suggest reusing an existing data, which is defined as `Voting.Vote`
It's simply a String wrapper, but
1. there's already a decoder defined for it, therefore we just need to `import Votes._` inside the `JsonDecoders.Circe.Operations` object definition and should be all set up
2. the existing decoder takes care to only correctly decode expected values (look at the definition in the JsonDecoders file
3. the votes are eventually stored as Strings in the table conversion, so we won't even need to pattern match, simply extract the string from the Votes value
4. it'll avoids different overlapping definition of a essentially the same concept (if we go for the enumeration encoding, I'd be keen to remove the existing one for a single way anyway)

I leave it to you to choose what you find best

Reply:
I think this isn't high priority enough to go through the effort, I'll use your Vote. Thank you!